libraries/AP_Camera/AP_Camera_Relay.cpp

```diff
@@ -14,11 +14,7 @@ void AP_Camera_Relay::update()
         if (ap_relay == nullptr) {
             return;
         }
-        if (_params.relay_on) {
-            ap_relay->off(0);
-        } else {
-            ap_relay->on(0);
-        }
+        ap_relay->set(AP_Relay_Params::FUNCTION::CAMERA, !_params.relay_on);
```

Comment:
This pattern precludes us having two relay cameras, it seems.  You might be in the second camera backend here.

Reply:
Your right, but this is the current behavior. We certainly could add a second camera relay function in the future.